routes/accounts.js

```diff
@@ -80,10 +81,22 @@ function accountRoutes (server, options, next) {
     handler: function (request, reply) {
       var sessionId = toBearerToken(request)
 
-      return accounts.findAll({
-        db: options.db,
-        bearerToken: sessionId,
-        include: request.query.include
+      admins.validateSession(sessionId)
+
+      .catch(function (error) {
+        if (error.name === dbErrors.MISSING_DOC.name) {
```

Comment:
@nolanlawson thanks for your response. My question is here... is there something I can compare against, or do you think it's sufficient just to compare against the status code? From what I saw, it looks like most of the routes in this project compare the error.message to a string constant. I am wondering what is the best solution for this project?

I tried to compare against the real error thrown from PouchDb, as you can see on this line, because I find it more descriptive, but this creates the dependency problem @gr2m and I discussed in this pull request.

In this project, do you think it better to add something to the `extras` to compare against the real error or just compare against a hard coded status code in order to [KISS](https://en.wikipedia.org/wiki/KISS_principle)?